Multisig/Logic/Models/UInt256String.swift

```diff
@@ -44,6 +48,17 @@ struct UInt256String: Hashable, Decodable {
     }
 }
 
+extension UInt256String: ExpressibleByStringLiteral {
+    init(stringLiteral value: StringLiteralType) {
+        if let uint256Value = UInt256(value) {
+            self = UInt256String(uint256Value)
+        } else {
+            assertionFailure("Invalid literal UInt256 value: \(value)")
```

Comment:
maybe this should be preconditionFailure, otherwise we might get a logical error in production

Reply:
done